tap_postgres/tests/conftest.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+import pytest
+from pytest_docker_tools import build, container, fetch
+
+postres_password = os.getenv("TAP_POSTGRES_PASSWORD", "mypass")
+postgres_image = fetch(repository="postgres:14.6")
+postgres_fixture = container(
```

Comment:
This spins up a container for you. Interesting I normally go with using the CI tool itself to spin up a container for testing. 

In the interest of getting this PR into the tap quickly can we remove this and make a seperate issue? Right now spinning up a container / postgres instance yourself and running pytest works (Not ideal but it works)

We should really get CI updated and in github actions at a minimum. This approach is interesting too, but I think weighing in on the right choice here doesn't help move this PR forward.

Reply:
Yeah totally fair, I'll remove. Sorry, this PR got overloaded. Will avoid doing too much at once in future.